sdks/python/apache_beam/io/restriction_trackers.py

```diff
@@ -126,6 +127,7 @@ def try_claim(self, position):
 
       return False
 
+  # TODO(SDF): Re-use try_split(0).
```

Comment:
What do you mean by that?

Reply:
`try_split(keep_fraction=0.0)` is a synonym for checkpoint, we should get rid of the redundancy. Removing comment until try_split is actually added.